mayant15/lotus
# Windows port

- Makes everything a git submodule, removing conan
- Fix compiler errors for MSVC
- Code quality fixes suggested by ReSharper
- Everything works as it should via dlls
- Also checked with clang on Windows, works as expected

**To Test:**
- Open it as a folder with Visual Studio, let it do its thing and import the CMake project
- Set the example executable as the startup project if required, and the build it
- You should see a platform with a box, and a skybox around it. WSAD and mouse move the camera.
